.github/main.workflow

```diff
@@ -46,14 +46,15 @@ action "Linters" {
   ]
   env = {
     PYTHON_VERSION = "3.7"
-    PIP_PACKAGES = "bandit flake8 mypy pylint ."
+    PIP_PACKAGES = ".[dev]"
   }
 }
 
 action "Tests" {
   uses = "swillner/actions/python-run@master"
   args = [
-    "pytest tests -r a --cov=openscm --cov-report=''"
+    "touch tests/__init__.py",
```

Comment:
@lewisjared I don't know why this was needed but all the tests are now passing so I'm going to move on with my life